tests/sec-linked-records/OPS/package.opf

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Package Documents</dc:coverage>
+    <dc:creator>Matthew Chan</dc:creator>
+    <dc:date>2021-10-21</dc:date>
+    <dc:description>
+        Reading System must process and display the title and creator metadata from the package document.
+        An ONIX 3.0 format linked metadata record exists, but contains neither title nor creator metadata.
+    </dc:description>
+    <dc:identifier id="pub-id">sec-linked-records</dc:identifier>
```

Comment:
Apologies, but as of https://github.com/w3c/epub-tests/pull/73, which changes our naming scheme *again* (second apology), this test should be pkg-link-record (or similar). Please rename, thanks!

Reply:
My recent commit (below) updates test id to "pkg-linked-records" per new scheme!
Ready for merge.